testsuites/syncgateway/functional/tests/conftest.py

```diff
@@ -144,6 +144,10 @@ def pytest_addoption(parser):
                      action="store_true",
                      help="delta-sync: Enable delta-sync for sync gateway")
 
+    parser.addoption("--magma-storage",
```

Comment:
this set default value to true. what if for non-magma version, will setting magma-storage to true break anything if server version is not magma? if non-magma version is with it, then nothing need to change. if there is something not compatible, then setting default value to false may be easier